sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -131,7 +131,8 @@ extern "C" {
  */
 #ifndef CONFIG_GNRC_IPV6_NIB_ADV_ROUTER
 #if CONFIG_GNRC_IPV6_NIB_ROUTER && \
-    (!CONFIG_GNRC_IPV6_NIB_6LR || CONFIG_GNRC_IPV6_NIB_6LBR)
+    (!CONFIG_GNRC_IPV6_NIB_6LR || CONFIG_GNRC_IPV6_NIB_6LBR) && \
+    !IS_USED(MODULE_GNRC_DHCPV6_CLIENT_SIMPLE_PD)
```

Comment:
Still have some head ache about adding a DHCPv6 dependency to the NIB... maybe we can just deactivate router advertisements in the initialization code or have a CFLAG set in the module Makefile (not sure how to do this in Kconfig though) instead of a compile time dependency here?

Reply:
made it a CFLAG.
There is no KConfig entry for `GNRC_DHCPV6_CLIENT_SIMPLE_PD` yet